app/jobs/remote_server_job.rb

```diff
@@ -96,6 +96,8 @@ def ruby_trunk_discourse(ssh, commit_hash, options)
   end
 
   def rails_releases(ssh, rails_version, options)
+    include_patterns = build_include_patterns(options[:include_patterns])
```

Comment:
The method is called in a few other methods but there isn't a need to assign it to a variable.

Date: Wed, 17 Jun 2015 09:18:33 -0700
From: notifications@github.com
To: ruby-bench-web@noreply.github.com
CC: tgx_world@hotmail.com
Subject: Re: [ruby-bench-web] Allow patterns to be passed to rails benchmarks (#91)

In app/jobs/remote_server_job.rb:

> @@ -96,6 +96,8 @@ def ruby_trunk_discourse(ssh, commit_hash, options)
>    end
>  
>    def rails_releases(ssh, rails_version, options)
> +    include_patterns = build_include_patterns(options[:include_patterns])

Do you want me to remove the build_include_patterns method and just call -e \"INCLUDE_PATTERNS=... directly


―
Reply to this email directly or view it on GitHub.